index.d.ts

```diff
@@ -1,7 +1,19 @@
 import * as React from 'react';
 
+export interface SourceSpec {
+  src: string;
+  type: string;
+}
+
+export interface TrackSpec {
```

Comment:
I think the names `SourceProps` and `TrackProps` make more sense?

Reply:
+1